scripts/integration-headless.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+CI=false npm run build && \
```

Comment:
what's CI=false?

Reply:
Travis treats warning message as error. So specifically for the build phase it allows warning message to pass. 

This is needed for some warning during the build

Reply:
https://github.com/facebook/create-react-app/issues/3657